infra/recipes/terraform/azure/sandbox/storage-services.tf

```diff
@@ -0,0 +1,7 @@
+resource "azurerm_storage_account" "sandbox" {
```

Comment:
Why did we not add network rules? Specifically, sandbox environments we can keep all the machines in the network in the same sandbox?

Reply:
Okay I added. Not fully sure if I did it right though, let me know if I should change anything